pyriemann/datasets/sampling.py

```diff
@@ -0,0 +1,340 @@
+from functools import partial
+import numpy as np
+from sklearn.utils import check_random_state
+from pyriemann.utils.base import sqrtm, expm
+
+
+def _pdf_r(r, sigma):
+    """pdf for the log of eigenvalues of a SPD matrix
+
+    Probability density function for the logarithm of the eigenvalues of a SPD
+    matrix samples from the Riemannian Gaussian distribution. See Said et al.
+    "Riemannian Gaussian distributions on the space of symmetric positive
+    definite matrices" (2017) for the mathematical details.
+
+    Parameters
+    ----------
+    r : ndarray, shape (n_dim,)
+        Vector defines in R^n_dim.
```

Comment:
Can you give a more precise description for `r`? Because `Vector defines in R^n_dim` is an information already given by `ndarray, shape (n_dim,)`.